src/lib/forms/core/DateTimeFields/DateInputField.js

```diff
@@ -20,6 +25,15 @@ export class DateInputField extends React.Component {
           handleBlur={props.form.handleBlur}
           handleDateChange={(value, name) => {
             props.onChange(fromISO(value), name);
+            if (_isEmpty(value)) {
+              if (fieldPath.includes('.')) {
+                let splittedFieldPath = fieldPath.split('.');
```

Comment:
I think you can deconstruct that to `let [keyPath, nestedKeyPath, ...parts] = fieldPath.split('.')`. Then, you can use the new variables below. I think it would make it easier to read. WDYT?

Reply:
Yes, I agree. Thanks!